js/game.js

```diff
@@ -0,0 +1,12 @@
+(function(){
+  var gameSection = document.getElementById('section');
+  var game = gameSection && gameSection.wordSearchGame();
+  var words = game.settings.wordsList,
+  hints = document.querySelector('.hints');            //Put words into `.hints`
+  words.forEach( word => {
+    var wordList = document.createElement('li');
```

Comment:
why we are creating the dom from js, create the list in html, and fill the list with js here